### DISCUSSION TITLE
Upgrade Mercurial to 6.2

### DISCUSSION THREAD

**Discussion by rgoldberg** [Homebrew/discussions#3445] [Writing Formulae/Casks] [answered]
It seems that completion file path issues must be fixed for Mercurial to be upgraded to 6.2 (see https://github.com/Homebrew/homebrew-core/pull/105540#issuecomment-1181569730).

I don't know the details of Homebrew's or Mercurial's handling of completions.

The existing PR is closed.

Where should work on this issue be tracked? The New Issue form indicates that issues can be opened for bugs. It doesn't mention opening an issue for upgrading to a new version, so I'm not sure if a new issue will get closed or will be left open.

**Comment by carlocab** [comment]
Work on version bumps is typically tracked in a PR. Issues usually are a very poor way of tracking them, so I don't think we'd keep one of those open.

**Reply by rgoldberg** [reply]
The PR was closed as stale, so the work isn't being tracked anywhere.

**Reply by carlocab** [reply]
It's still accessible, can be commented on, and shows the complete state of work done.

A new PR can be opened if and when additional work is done.

**Reply by rgoldberg** [reply]
Does it show up anywhere as something that needs work? So homebrew maintainers looking to work on things know that it is still needs work? I'd fix it, but I don't know anything about home-brew completion script handling, don't program in python, etc.

**Reply by carlocab** [reply, marked as answer]
> Does it show up anywhere as something that needs work? So homebrew maintainers looking to work on things know that it is still needs work?

It shows up in your post here and for users of `brew livecheck`. The aforementioned users include maintainers and contributors who regularly update formulae in homebrew-core.

**Reply by ramoncerda2025** [reply]
https://photos.app.goo.gl/upjb96cxuZWGaaLPA
